frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx

```diff
@@ -306,6 +306,9 @@ export function InsightsTable({
         })
     }
 
+    // don't use URL for sorting when creating an insight
+    const useURLForSorting = !!insightProps.dashboardItemId && !insightProps.dashboardItemId.startsWith('new')
+
```

Comment:
I think the overall solution makes sense but I wonder if here it makes more sense to use the "editing" mode as the flag rather than the "new" check (if that is possible) as otherwise this remains broken when editing an Insight